### ISSUE TITLE
Add timestamp to name of spinnaker directory created when uploading files to Core

### ISSUE THREAD

**Issue by klearned** [BD2KGenomics/dcc-feature-requests#2]
**Problem:**
When I need to re-upload fixed files, I am often doing so from a directory I've previously uploaded from and I keep forgetting to rename the spinnaker directory so that I don't get this issue: https://github.com/BD2KGenomics/core-operations/issues/100

**Suggestion:**
Add timestamp or date to spinnaker directory name when it is created so I don't have to remember to rename the spinnaker directory if I have to do re-uploads. That will also help me keep track of my previous spinnaker uploads because I will know when I did them at a glance.

So, the spinnaker directory would be something like:

spinnaker_2017-04-13
or
spinnaker_2017-04-13_00:00:00

**Comment by klearned** [comment]
@wshands @benjaminran 
Another problem with the current situation...This morning I wanted to upload more fixed files, so I renamed the spinnaker directory. However, I didn't realize that Meghan was in the middle of an upload from that same directory, and I renamed her spinnaker directory durning her upload.

Will this cause problems? The files she was uploading are on the File Browser. She didn't receive any errors on the command line, but the spinnaker.log is split between two files in two diff directories. Hmm, what about the metadata spinnaker writes to the output_metadata directory? Does it use that info in the upload?

If there had been a timestamp in the spinnaker directory name, this would have been prevented.
